python/ray/rllib/evaluation/policy_evaluator.py

```diff
@@ -718,6 +718,7 @@ def _validate_env(env):
 
 
 def _monitor(env, path):
+    from gym import wrappers
```

Comment:
is this line needed?

Reply:
since i couldn't run our env inside _monitor wrapper it isn't. (but the method throws an error if called)